WeatherService.Tests/ParsersTesting/ConfigServiceTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Moq;
+using WeatherService.Models;
+using WeatherService.Services.FileReaders;
+using FluentAssertions;
+
+namespace WeatherService.Tests.ParsersTesting;
+
+public class ConfigServiceTests
+{
+    [Fact]
+    public void LoadConfigSettings_ShouldReturnDeserializedDictionary()
+    {
+        var fakeJsonConfig = """
+            {
+            "RainBot": {
+              "Enabled": true,
+              "Threshold": 70,
+              "Message": "It looks like it's about to pour down!"
+              }
+            }                
+            """;
+        var mockFileReader = new Mock<IFileReader>();
+        mockFileReader.Setup(x => x.ReadAllText(It.IsAny<string>())).Returns(fakeJsonConfig);
+        var configService = new ConfigService("botsSettings.json", mockFileReader.Object);
+
+
+        var results = configService.loadConfigSettings();
+
+
+        results.Should().ContainKey("RainBot");
+        results["RainBot"].Enabled.Should().Be(true);
+        results["RainBot"].Threshold.Should().Be(70);
+        results["RainBot"].Message.Should().Be("It looks like it's about to pour down!");
+
```

Comment:
1. Explicitly define the test method sections: AAA (Arrange, Act and Assert).
2. Preserve consistent line spaces between the different sections.
3. Use constant when it's possible in the Arrange section. 

Same thing for all other test methods in this PR.

```suggestion
        // Arrange
const string fakeJsonConfig = """
    {
    "RainBot": {
      "Enabled": true,
      "Threshold": 70,
      "Message": "It looks like it's about to pour down!"
      }
    }                
    """;

var mockFileReader = new Mock<IFileReader>();
mockFileReader.Setup(x => x.ReadAllText(It.IsAny<string>())).Returns(fakeJsonConfig);
var configService = new ConfigService("botsSettings.json", mockFileReader.Object);

// Act
var results = configService.loadConfigSettings();

// Assert
results.Should().ContainKey("RainBot");
results["RainBot"].Enabled.Should().Be(true);
results["RainBot"].Threshold.Should().Be(70);
results["RainBot"].Message.Should().Be("It looks like it's about to pour down!");
```